### ISSUE TITLE
Support für  Homematic IP  Türschlossantrieb HmIP-DLD

### ISSUE THREAD

**Issue by bjoerngiese93** [thkl/homebridge-homematic#676]

**Comment by Ritschi87** [comment]
Gibt es hierzu schon was neues?

**Comment by GUNAU** [comment]
Wird den der alte Homematic Funk-Türschlossantrieb KeyMatic : HM-Sec-Key unterstützt ?

**Comment by bjoerngiese93** [comment]
Ja er wird unterstützt￼

**Comment by 011V32** [comment]
@thkl Wird dieses Prejekt noch gewartet oder sollen wir auf das [hap-homematic](https://github.com/thkl/hap-homematic) Projekt portieren?

Wäre es eventuell denkbar die Core-Funktionen auszulagern und von beiden Projekten zu nutzen?
(Damit immer beide Projekte Up2Date sind und weniger Aufwand entsteht.)

Danke.

**Comment by thkl** [comment]
Ein Sharing der Serviceklassen wäre möglich, dazu müsste ich aber dieses Projekt hier komplett redesignen. 

Da hap-Homematic inzwischen durch die Anpassung auf debmatic auch außerhalb der CCU laufen kann ist das eigentlich doppelte Arbeit. 

Das hap-Homematic Projekt wird auf jeden Fall von mir besser Supportet als dieses hier.

**Comment by Oli380** [comment]
wird jetzt der HmIP-DLD unterstützt oder eher nicht mehr?

**Comment by juangamnik** [comment]
I am unsure whether I have to move to hap-homematic (I use debmatic on a raspberry pi and there seems to be no possibility to install hap-homematic...). Or is/will HmIP-DLD be supported by this project?

**Comment by 011V32** [comment]
It is possible with debmatic.
See [https://github.com/thkl/hap-homematic/blob/master/doc/debmatic.md](https://github.com/thkl/hap-homematic/blob/master/doc/debmatic.md).
But for me it was too buggy so i switched to raspberrry matic which is much better now.

**Comment by juangamnik** [comment]
Yeah I used this guide, but get TypeErrors… opened an issue there.
